accumulo-handler/src/java/org/apache/hadoop/hive/accumulo/serde/AccumuloSerDe.java

```diff
@@ -54,6 +54,11 @@ public class AccumuloSerDe extends AbstractSerDe {
 
   private static final Logger log = LoggerFactory.getLogger(AccumuloSerDe.class);
 
+  @Override
+  public Logger getClassLogger() {
+    return log;
+  }
+
```

Comment:
How can I avoid this pattern and still achieve the instantiated class name in the log for debugging/reading purposes. Like I see below in the log:
**orc.OrcSerde: SerDe initialized: [Optional[Configuration:.....**
If I do not override method to return this fields and rather use the parent class logger then I will end up something like below:
**serde2.AbstractSerDe: SerDe initialized: [Optional[Configuration:.....**

All this is for the one line of log in https://github.com/apache/hive/blob/cbd6363ce904ea9150a8e8591d60edfc6b4adea9/serde/src/java/org/apache/hadoop/hive/serde2/AbstractSerDe.java#L93

Reply:
How about the following?
```java
LOG.debug("{} initialized: [{}][{}]", getClass().getName(), this.configuration, this.properties);
```
It will generate something like:
```
serde2.AbstractSerDe: OrcSerde initialized ...
```
which I believe reflects better what is happening. The message is logged in `AbstractSerDe` class but the actual object is of `OrcSerde` type.

Reply:
Yes this will be better, I will change the patch accordingly